components/server/src/workspace/workspace-starter.ts

```diff
@@ -931,6 +931,7 @@ export class WorkspaceStarter {
             req.setSource(src);
             req.setAuth(auth);
             req.setForceRebuild(forceRebuild);
+            req.setTriggeredBy(user.id);
```

Comment:
Note that it's not defined what the "real owner" of an image build is, exactly. Calling it "triggered by" though makes sense.